packages/psss/src/containers/Modals/DeleteAlertModal.js

```diff
@@ -31,20 +31,25 @@ export const DeleteAlertModal = ({ isOpen, handleClose, alertId }) => {
     setStatus(STATUS.SUCCESS);
   }, [deleteAlert]);
 
+  const onClose = useCallback(async () => {
+    setStatus(STATUS.INITIAL);
+    handleClose();
+  }, [setStatus, handleClose]);
+
   if (status === STATUS.SUCCESS) {
     return (
       <SuccessModal
         isOpen={isOpen}
         title="Delete Alert"
         mainText="Alert successfully deleted"
-        handleClose={handleClose}
+        handleClose={onClose}
```

Comment:
A common convetion in React is to name the prop `on{{Event}}` and the handler `handle{{Event}}`:

```suggestion
        onClose={handleClose}
```

I could have commented this in an earlier PR but I just noticed

Reply:
I can never say no to following convention 😬 . However I noticed there are other places still doing this, but I've just changed what I've touched 👍

Reply:
Thanks for taking the time to do so! 💯